source/templates.tex

```diff
@@ -623,7 +623,6 @@
 \grammarterm{template-parameter}.
 \begin{example}
 \begin{codeblock}
-template <class T = float> struct B {};
```

Comment:
This just seems to be the removal of an unused identifier.

Reply:
Yes - it simply distracts from the example.